topics/go/algorithms/strings/reverse/reverse_test.go

```diff
@@ -0,0 +1,29 @@
+package strings_test
+
+import (
+	strings "github.com/ardanlabs/gotraining/topics/go/algorithms/strings/reverse"
+	"testing"
+)
+
+const succeed = "\u2713"
+const failed = "\u2717"
+
+func TestReverseString(t *testing.T) {
+
+	revTests := []struct {
+		name   string
+		input  string
+		output string
```

Comment:
Maybe call this expected? What do you think?

Reply:
Sounds good.